sdk/identity/azure-identity/test/ut/azure_pipelines_credential_test.cpp

```diff
@@ -644,7 +644,7 @@ TEST(AzurePipelinesCredential, InvalidServiceConnectionId_LIVEONLY_)
   }
 }
 
-TEST(AzurePipelinesCredential, InvalidSystemAccessToken_LIVEONLY_)
+/*TEST(AzurePipelinesCredential, InvalidSystemAccessToken_DISABLED_)
```

Comment:
Why is this commented out?

Reply:
somehow DISABLE does not disample this test

Reply:
`DISABLED_` needs to be a **prefix** of the test name, not suffix.